pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -78,7 +77,7 @@ def _kokkos_configuration():
 allowed_operations = {
     "Identity",
     "BasisState",
-    "QubitStateVector",
+    "StatePrep",
```

Comment:
```suggestion
   "QubitStateVector",
    "StatePrep",
```

if you do `qml.QubitStateVector(array, wires)`, the operation's name is still `QubitStateVector`.

Reply:
Thanks for pointing this out, I thought `QubitStateVector` was fully deprecated.